functional-tests/src/test/java/com/redhat/che/functional/tests/RhCheAbstractTestClass.java

```diff
@@ -74,4 +82,12 @@ public void closeFilesAndProject() {
   public void closeBrowser() {
     ide.close();
   }
+
+  public boolean isProd() {
+    return cheHost.equals(CHE_PROD_URL);
```

Comment:
I would change it to CHE_PROD_URL.equals(cheHost) to avoid NPE

Reply:
both variables must always be set. if they are not the test would never run and fail.

Reply:
refactored.